CalSmokeApp/bin/appcenter.sh

```diff
@@ -34,21 +34,10 @@ BUILD_RUN_LOOP: iff 1, then rebuild Calabash iOS gem before uploading.
   exit 64
 fi
 
-CREDS=.appcenter-credentials
-if [ ! -e "${CREDS}" ]; then
-  error "This script requires a ${CREDS} file"
-  error "Generating a template now:"
-  cat >${CREDS} <<EOF
-export APPCENTER_TOKEN=
-EOF
-  cat ${CREDS}
-  error "Update the file with your credentials and run again."
-  error "Bye."
-  exit 1
```

Comment:
In my understanding we shouldn't disallow to run this script locally. I suggest to implement the following:
- Check for `.appcenter-credentials` file. If it exists - load credentials from it.
- If `.appcenter-credentials` does not exist (perhaps this script performs on CI) - take credentials from `find-keychain-credential.sh`

Reply:
Added check file with credentials.